### ISSUE TITLE
V-71943: Profile Errors

### ISSUE THREAD

**Issue by HackerShark** [mitre/redhat-enterprise-linux-7-stig-baseline#31]
This control is throwing profile errors in the inspec results

**Comment by ljkimmel** [comment]
@trevor-vaughan 
@aaronlippold  said you might have some input on this one...
The errors mentioned in the issue are caused because we missed copying over the variable declarations:
``` ruby
  required_rules = [
    'auth required pam_faillock.so unlock_time=.*',
    'auth sufficient pam_unix.so try_first_pass',
    'auth [default=die] pam_faillock.so unlock_time=.*'
  ]
  alternate_rules = [
    'auth required pam_faillock.so unlock_time=.*',
    'auth sufficient pam_sss.so forward_pass',
    'auth sufficient pam_unix.so try_first_pass',
    'auth [default=die] pam_faillock.so unlock_time=.*'
  ]
```
But that brings up the question of whether that is being too rigid since PAM is so highly configurable. In any case, I think, `pam_faillock` is always followed by `preauth`, `authfail` or `authsucc` so, if we match those lines exactly it would never match since `pam_faillock.so` is followed immediately by `unlock_time`. 
Long story short, we should confer on how to handle this and other PAM controls.

**Comment by aaronlippold** [comment]
Hi,

I think the best way to get moving forward on this is to chat about it on
the SIMP slack channel.

That way we can address it directly.

Thanks,

Aaron
--------
Aaron Lippold

lippold@gmail.com

260-255-4779

twitter/aim/yahoo,etc.
'aaronlippold'

**Comment by trevor-vaughan** [comment]
@ljkimmel Honestly, the PAM rules are a huge pain to deal with correctly.

The SSG STIG checks are WAY too rigid but allowing people to jump required rules is also bad so this was kind of the happy medium that I came up with.
